lib/src/ui/pages/home/home_page.dart

```diff
@@ -1,248 +1,38 @@
 import 'package:allthenews/generated/l10n.dart';
-import 'package:allthenews/src/di/injector.dart';
-import 'package:allthenews/src/domain/model/article.dart';
-import 'package:allthenews/src/ui/common/util/date_time_extensions.dart';
-import 'package:allthenews/src/ui/common/util/dimens.dart';
-import 'package:allthenews/src/ui/common/util/notifier_state.dart';
-import 'package:allthenews/src/ui/common/util/untranslatable_strings.dart';
-import 'package:allthenews/src/ui/common/widget/primary_icon_button.dart';
-import 'package:allthenews/src/ui/common/widget/primary_text_button.dart';
-import 'package:allthenews/src/ui/pages/home/home_notifier.dart';
-import 'package:allthenews/src/ui/pages/home/home_notifier_state.dart';
-import 'package:allthenews/src/ui/pages/home/news/news_list_page.dart';
-import 'package:allthenews/src/ui/pages/home/news/primary_news/primary_news_list_entity.dart';
-import 'package:allthenews/src/ui/pages/home/news/primary_news/primary_news_list_view.dart';
-import 'package:allthenews/src/ui/pages/home/news/secondary_news/secondary_news_list_entity.dart';
-import 'package:allthenews/src/ui/pages/home/news/secondary_news/secondary_news_list_item.dart';
-import 'package:allthenews/src/ui/pages/settings/settings_notifier.dart';
-import 'package:allthenews/src/ui/pages/settings/settings_page.dart';
+import 'package:allthenews/src/ui/pages/feed/feed_page.dart';
+import 'package:allthenews/src/ui/pages/profile/profile_page.dart';
 import 'package:flutter/material.dart';
-import 'package:provider/provider.dart';
-
-abstract class _Constants {
-  static const appBarActionsVerticalPadding = 11.0;
-  static const appBarActionsIconsPadding = 8.0;
-  static const appBarTitleFontFamily = 'Chomsky';
-  static const appBarTitleLeftPadding = 10.0;
-  static const sectionHeaderPadding = 10.0;
-  static const sectionSpacing = 20.0;
-  static const primaryNewsListSize = 5;
-  static const retryButtonWidth = 120.0;
-  static const retryButtonHeight = 40.0;
-}
 
 class HomePage extends StatefulWidget {
   @override
   _HomePageState createState() => _HomePageState();
 }
 
 class _HomePageState extends State<HomePage> {
-  final HomeNotifier _homeNotifier = inject<HomeNotifier>();
-
-  @override
-  void initState() {
-    super.initState();
-    _homeNotifier.fetchHomeArticles();
-  }
-
-  @override
-  void dispose() {
-    super.dispose();
-    _homeNotifier.dispose();
-  }
+  var _selectedPage = 0;
+  final _pages = [
```

Comment:
jak przechodzę do ustawień, a następnie z ustawień wychodzę strzałką wstecz na toolbarze to wraca mi na ekran FeedPage, ale nie wyświetla navigationbara